src/bundle-configuration/zookeeper-prod/runtime-config.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+export ZK_DATA_DIR=/var/lib/zookeeper
```

Comment:
@huntc Sure, `/var/lib` should stay the same. The questions is if have one of the two conventions for prod bundles:

- /var/lib/zookeeper-v1
- /var/lib/cassandra-v1
- /var/lib/elasticsearch-v1

OR

- /var/lib/zookeeper
- /var/lib/cassandra
- /var/lib/elasticsearch